modules/react/modal/lib/ModalCard.tsx

```diff
@@ -18,7 +18,8 @@ const ResponsiveModalCard = styled(Popup.Card)<ModalCardProps & StyledType>(({th
   padding: space.l,
   [theme.canvas.breakpoints.down('s')]: {
     margin: `0 0 ${space.xl}`,
-    padding: space.m,
+    padding: space.s,
+    borderRadius: 24, // matches mobile specs
```

Comment:
I could use our spacing token, but our `borderRadius` token only goes up to `8px` which is why i have a hard coded 24. I'm happy to change this to use our spacing token or update our borderRadius to have a new value

Reply:
This is the second time I've heard us discussing spacing tokens and them not being aligned with mobile/smaller screens. Should we have a larger discussion about this to establish a better approach?

Reply:
Sorry I should have clarified. `space.m` gives us `24px` but it felt strange to use that here since we're styling `borderRadius` and our `borderRadius` type only goes to `8px`